rpc/send.go

```diff
@@ -69,6 +69,11 @@ type rpcError struct {
 func (r rpcError) Error() string { return r.errMsg }
 
 // CanRetry implements the Retryable interface.
+// TODO(tschottdorf): the way this is used by rpc/send suggests that it
```

Comment:
This depends on which level you're looking at: we may not want to retry them internally at the DistSender, but if an rpc error is returned to the client the client would be able to retry.